### PR TITLE
📖 Update generating crd link in docs

### PR DESCRIPTION
**Pull request by pravarag**
Fixes: https://github.com/kubernetes-sigs/kubebuilder/issues/2276

The [Generating CRDs](https://github.com/kubernetes-sigs/kubebuilder/blob/cb1b6008f16ad5b4f56b019600f285e902215795/docs/book/src/reference/generating-crd.md) link is broken, a small correction to fix it.

### CODE REVIEW

File: docs/book/src/reference/markers/crd.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ These markers describe how to construct a custom resource definition from
 a series of Go types and packages.  Generation of the actual validation
 schema is described by the [validation markers](./crd-validation.md).
 
-See [Generating CRDs](/reference/generating-crd.md) for examples.
+See [Generating CRDs](/docs/book/src/reference/generating-crd.md) for examples.
```
Comment (estroz) [inline docs/book/src/reference/markers/crd.md:7]: I don't see this prefix being used elsewhere in docs. I'd say just use relative pathing
```suggestion
See [Generating CRDs](../generating-crd.md) for examples.
```

Comment (pravarag) [reply docs/book/src/reference/markers/crd.md:7]: Thanks @estroz , I'll update as suggested.

Comment (pravarag) [reply docs/book/src/reference/markers/crd.md]: Done 🙂

**Comment by estroz** [comment]
/approve
/lgtm

Thanks @pravarag

**Pull request merged**